samples/nrf9160/https_client/src/main.c

```diff
@@ -167,20 +168,25 @@ void main(void)
 		printk("Failed to connect to the LTE network, err %d\n", err);
 		return;
 	}
-	printk("OK\n");
```

Comment:
Why was this line removed?

Reply:
Just FYI, the CI failure originates here - the CI output parsing script expects this OK.

Reply:
Originally because I replaced it with a printk that output the details of the next step (calling getaddrinfo()), but I now have an snprintf inbetween. I've put it back in.